docs/resources/Webhook.md

```diff
@@ -0,0 +1,126 @@
+	}
+}
+```
+
+## Create Webhook % POST /channels/{channel.id#DOCS_CHANNEL/channel-object}/webhooks
+
+Create a new webhook. Returns a [webhook](#DOCS_WEBHOOK/webhook-object) object on success.
+
+###### JSON Params
+
+| Field | Type | Description |
+|-------|------|-------------|
+| name | string | name of the webhook (2-100 characters) |
+| avatar | string | base64 128x128 jpeg image for the default webhook avatar |
+
+## Get Channel Webhooks % GET /channels/{channel.id#DOCS_CHANNEL/channel-object}/webhooks
+
+Returns a list of channel [webhook](#DOCS_WEBHOOK/webhook-object) objects.
+
+## Get Guild Webhooks % GET /guilds/{guild.id#DOCS_GUILD/guild-object}/webhooks
+
+Returns a list of guild [webhook](#DOCS_WEBHOOK/webhook-object) objects.
+
+## Get Webhook % GET /webhooks/{webhook.id#DOCS_WEBHOOK/webhook-object}
+
+Returns the new [webhook](#DOCS_WEBHOOK/webhook-object) object for the given id.
+
+## Get Webhook with Token % GET /webhooks/{webhook.id#DOCS_WEBHOOK/webhook-object}/{webhook.token#DOCS_WEBHOOK/webhook-object}
+
+Same as above, except this call does not require authentication and returns no user in the webhook object.
+
+## Modify Webhook % PATCH /webhooks/{webhook.id#DOCS_WEBHOOK/webhook-object}
+
+Modify a webhook. Returns the updated [webhook](#DOCS_WEBHOOK/webhook-object) object on success.
+
+>info
+> All parameters to this endpoint are optional
+
+###### JSON Params
+
+| Field | Type | Description |
+|-------|------|-------------|
+| name | string | the default name of the webhook |
+| avatar | string | base64 128x128 jpeg image for the default webhook avatar |
+
+## Modify Webhook with Token % PATCH /webhooks/{webhook.id#DOCS_WEBHOOK/webhook-object}/{webhook.token#DOCS_WEBHOOK/webhook-object}
+
+Same as above, except this call does not require authentication and returns no user in the webhook object.
+
+## Delete Webhook % DELETE /webhooks/{webhook.id#DOCS_WEBHOOK/webhook-object}
+
+Delete a webhook permanently. User must be owner. Returns a 204 NO CONTENT response on success.
+
+## Delete Webhook with Token % DELETE /webhooks/{webhook.id#DOCS_WEBHOOK/webhook-object}/{webhook.token#DOCS_WEBHOOK/webhook-object}
+
+Same as above, except this call does not require authentication.
+
+## Execute Webhook % POST /webhooks/{webhook.id#DOCS_WEBHOOK/webhook-object}/{webhook.token#DOCS_WEBHOOK/webhook-object}
+
+>warn
+> This endpoint supports both JSON and form data bodies. It does require multipart/form-data requests instead of the normal JSON request type when uploading files. Make sure you set your `Content-Type` to `multipart/form-data` if you're doing that.
+
+###### Querystring Params
+
+| Field | Type | Description | Required |
+|-------|------|-------------|----------|
+| wait | bool | waits for server confirmation of message send before response, and returns the created message body (defaults to `false`; when `false` a message that is not saved does not return an error) | false |
+
+###### JSON/Form Params
```

Comment:
we really need more examples for this. I'd love to do what slack does. But I think it's mergable as is.